eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -229,7 +229,7 @@ jobs:
       testGroup: innerloop
       nameSuffix: AllSubsets_Mono
       isExtraPlatforms: ${{ parameters.isExtraPlatformsBuild }}
-      buildArgs: -s mono+libs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true /p:DevTeamProvisioning=- /p:RunAOTCompilation=true $(_runSmokeTestsOnlyArg) /p:BuildTestsOnHelix=true /p:EnableAdditionalTimezoneChecks=true /p:UsePortableRuntimePack=true /p:BuildDarwinFrameworks=true
+      buildArgs: -s mono+libs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true /p:DevTeamProvisioning=- /p:RunAOTCompilation=true $(_runSmokeTestsOnlyArg) /p:BuildTestsOnHelix=true /p:EnableAdditionalTimezoneChecks=true /p:UsePortableRuntimePack=true /p:BuildDarwinFrameworks=true /p:IsManualOrRollingBuild=true
```

Comment:
Is `IsManualOrRollingBuild` primarily to make the three test suites 
`System.IO.MemoryMappedFiles.Tests.csproj`, `System.Runtime.Numerics.Tests.csproj`, and `System.Formats.Cbor.Tests.csproj` run for iOS device?
Don't the iOS/tvOS device tests only run for rolling/manual (I'm assuming manual = runtime-extra-platforms) already? If thats true, this prop *feels* superfluous.

In other words, is this analogous to removing `/p:IsManualOrRollingBuild=true` and merging 
```
  <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetOS)' == 'iOS'">
    <!-- Only System.Runtime tests on iOS for now -->
    <ProjectReference Include="$(MSBuildThisFileDirectory)System.Runtime\tests\System.Runtime.Tests.csproj" />

    <ProjectReference Include="$(RepoRoot)\src\tests\FunctionalTests\iOS\Device\**\*.Test.csproj"
                      Exclude="@(ProjectExclusions)"
                      BuildInParallel="false" />
  </ItemGroup>

  <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetOS)' == 'iOS' and '$(IsManualOrRollingBuild)' == 'true'">
    <!-- These crash on tvOS, but do not on iOS. Run these only on the rolling builds -->
    <ProjectReference Include="$(MSBuildThisFileDirectory)System.IO.MemoryMappedFiles\tests\System.IO.MemoryMappedFiles.Tests.csproj" />
    <ProjectReference Include="$(MSBuildThisFileDirectory)System.Runtime.Numerics\tests\System.Runtime.Numerics.Tests.csproj" />
    <ProjectReference Include="$(MSBuildThisFileDirectory)System.Formats.Cbor\tests\System.Formats.Cbor.Tests.csproj" />
  </ItemGroup>
  ```

Reply:
I found an interesting problem that we haven't encountered in that the 3 suites grouped here routinely fail on tvOS runs, but pass on iOS.  Previously, iOS was configured to only run smoke tests (System.Runtime), but I felt to get *some* coverage, I'd include them in runs that were triggered manually or scheduled.

We do not run full iOS device tests due to not having enough capacity.

Reply:
We already do have enough iPhone capacity btw (`osx.1200.amd64.iphone.open` has 50)

Reply:
>  I'd include them in runs that were triggered manually or scheduled.

Right, there aren't any runs for iOS device that aren't rolling or triggered manually are there?

> We do not run full iOS device tests due to not having enough capacity.

Yeah, that makes sense, and so we only run System.Runtime. I'm wondering if it makes more sense to just bundle `System.IO.MemoryMappedFiles.Tests.csproj, System.Runtime.Numerics.Tests.csproj, and System.Formats.Cbor.Tests.csproj` with System.Runtime.

Reply:
I think I'm going to leave this as is.  The original purpose of running smoke tests on PR's was to reduce the total work time on 'heavyweight' runs like iOS.